stellargraph/core/graph_networkx.py

```diff
@@ -646,31 +646,26 @@ def create_graph_schema(self, create_type_maps=True, nodes=None):
         """
 
         if nodes is None:
-            nodes = self._graph.nodes()
-            edges = self._graph.edges(keys=True)
+            nodes = self.nodes()
+            edges = self.edges(triple=True)
 
         elif create_type_maps is False:
-            edges = self._graph.edges(nodes, keys=True)
+            edges = self.edges(triple=True)
```

Comment:
Hmm previously, `self._graph.edges(nodes, keys=True)` would only collect edges that are incident on `nodes`, whereas now I think it's collecting all edge triples? This `elif` path is when `nodes` is not `None`, and so I think we still want preserve the original behaviour?

It would effect the resulting `edge_types` set that gets passed into constructing the `GraphSchema`

Reply:
I've switched back to a traversal over `self._graph.edges`.